sim/validator.py

```diff
@@ -56,6 +56,8 @@ def simulations(
         sample.simulate_genome(reference_path, simulated_genome_path + sample.name)
         sample.simulate_reads(simulated_genome_path, simulated_reads_path)
 
+    return simulated_reads_path
```

Comment:
I think it would be better to pass in the reads path rather than generate it in this function. Can be a `TODO` if you prefer: